package.json

```diff
@@ -42,8 +42,9 @@
     "docs": "node docs/build.js && docsify serve ./docs",
     "benchmark": "yarn build && node --experimental-specifier-resolution=node ./benchmark/run.js",
     "lint": "eslint --ext .js,.mjs,.ts .",
-    "prepack": "yarn build",
-    "prepare": "husky install",
+    "postinstall": "yarn clean-modules -f .yarnclean -y && husky install",
+    "postpack": "pinst --enable",
```

Comment:
I think we need to install this package in our dev dependencies, right?

Reply:
Nice catch!! The docs didn't mention it as an additional npm repo, and I just assumed it was some npm _magic_ 😅